magick-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Wrapper script for the 'convert' command to mimic 'magick'
```

Comment:
BTW perhaps we can make this script to live in `scripts` directory which is the usual convention for adding hepler scripts in other modules? (See r-client, r-server, r-w-controller...)